Sync/Synchronizers/ClientsSynchronizer.swift

```diff
@@ -380,12 +381,16 @@ open class ClientsSynchronizer: TimestampedSingleCollectionSynchronizer, Synchro
                             _ = command.run(self)
                         }
                         self.lastFetched = responseTimestamp!
+                        if isFirstSync,
+                           let notifier = notifier {
+                            DispatchQueue.main.async { _ = notifier.notifyAll(collectionsChanged: ["clients"]) }
```

Comment:
Any particular reason for doing this on the main thread?

Consider using the `syncQueue` or another background queue instead.